lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -3500,6 +3507,10 @@ getNetworkInformation st nl = liftIO $ do
         , Api.nodeTip = apiNodeTip
         , Api.networkTip = fst <$> nowInfo
         , Api.nodeEra = toApiEra nodeEra
+        , Api.networkInfo =
+            Api.ApiNetworkInfo
+                (if nid == Cardano.Mainnet then "mainnet" else "testnet")
```

Comment:
no ToText instance present for this? If yes, why not to use it?

Reply:
There is none that fits. This is only a way to json encode the protocol magic in the Api in 2 fields.